fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/properties/CardinalitiesProperty.java

```diff
@@ -368,6 +369,12 @@ public Cardinalities visitFullUnorderedScanExpression(@Nonnull final FullUnorder
         return Cardinalities.unknownCardinalities();
     }
 
+    @Nonnull
+    @Override
+    public Cardinalities visitGroupByExpression(@Nonnull final GroupByExpression element) {
+        return Cardinalities.unknownCardinalities();
+    }
+
```

Comment:
Can you check, and if needed update the implementation for streaming aggs here as well?

Reply:
Yes, there was a stub implementation for `CardinalitiesProperty#visitRecordQueryStreamingAggregationPlan` that returns, similar to the above `Cardinalities.unknownCardinalities()`. I modified it in a similar way to the above suggestion for `GroupByExpression`.